### ISSUE TITLE
Unable to install classInt on RStudio on Amazon AWS 

### ISSUE THREAD

**Issue by kamaulindhardt** [r-spatial/classInt#29]
**Hi classInt team and enthusiastist,**

Lately, I have been experimenting with moving some of my computation into [Amazon's AWS EC2](https://aws.amazon.com/premiumsupport/knowledge-center/ec2-instance-access-s3-bucket/?sc_channel=EL&sc_campaign=Demo_Deep_Dive_2021_vid&sc_medium=YouTube&sc_content=Video9558&sc_detail=STORAGE&sc_country=US) platform running RStudio, as it gives flexibility and enormous computer power. Unfortunately, I have several difficulties installing one of my favorite packages, `sf` because `classInt `is giving me some issues.

The original problem I had was with the sf package, so please see my issue [#1778](https://github.com/r-spatial/sf/issues/1778) on **r-spatial/sf** 

Any idea on this issue what might be going on. Have you had more experiences with this kind of issue for the `classInt` package on AWS EC2? Do you have any recommendations for me? Thank you!

Here is the error warning I get from RStudio in AWS EC2

`Restarting R session...`

```
> ?e1071
No documentation for ‘e1071’ in specified packages and libraries:
you could try ‘??e1071’
> install.packages(classInt)
Error in install.packages : object 'classInt' not found
```

```
> install.packages("calssInt")
Installing package into ‘/home/rstudio/R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘calssInt’ is not available (for R version 4.0.2)
```

```
> install.packages('classInt')
Installing package into ‘/home/rstudio/R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
also installing the dependency ‘e1071’

trying URL 'https://cran.rstudio.com/src/contrib/e1071_1.7-8.tar.gz'
Content type 'application/x-gzip' length 581347 bytes (567 KB)
==================================================
downloaded 567 KB

trying URL 'https://cran.rstudio.com/src/contrib/classInt_0.4-3.tar.gz'
Content type 'application/x-gzip' length 403884 bytes (394 KB)
==================================================
downloaded 394 KB

ERROR: failed to lock directory ‘/home/rstudio/R/x86_64-pc-linux-gnu-library/4.0’ for modifying
Try removing ‘/home/rstudio/R/x86_64-pc-linux-gnu-library/4.0/00LOCK-e1071’
Warning in install.packages :
  installation of package ‘e1071’ had non-zero exit status
ERROR: dependency ‘e1071’ is not available for package ‘classInt’
* removing ‘/home/rstudio/R/x86_64-pc-linux-gnu-library/4.0/classInt’
Warning in install.packages :
  installation of package ‘classInt’ had non-zero exit status

The downloaded source packages are in
	‘/tmp/RtmpKoPiK0/downloaded_packages’

```

All the best,

Kamau Lindhardt

**Comment by rsbivand** [comment]
Please stay with the original issue. No idea. never use clouds. You have a system problem, not a **classInt** problem. Try removing or avoiding rstudio, and install **e1071** from base R running in a regular terminal. This smells like a permissions problem, so your system problem. Also try a different mirror, some mirrors get corrupted images of packages.

**Comment by kamaulindhardt** [comment]
Thank you @rsbivand ,
Sorry.. 

I will contact the Amazon AWS web service team to understand what's going on with the system. I would avoid changing mirror/image as that would mean too many alterations

**Comment by kamaulindhardt** [comment]
I managed to solve it with the help of Edzar Pebesma of the`sf` package  

see my now closed issue [#1778](https://github.com/r-spatial/sf/issues/1778) on **r-spatial/sf**
